### PR TITLE
User Sing Up Page

### PR DESCRIPTION
**Pull request by Parastoo-S**
Making changes to the user login page

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,8 +10,10 @@ def create
 
     respond_to do |format|
       if @new_user.save
+        flash[:notice] = "Profile created successfully."
+        MsrMailer.welcome_email(@user).deliver_later
```
Comment (KarimChukfeh) [inline app/controllers/users_controller.rb]: @user doesn't exist here
you want to pass @new_user to welcome_email like so:
**MsrMailer.welcome_email(@new_user).deliver_later**

Comment (Parastoo-S) [inline app/controllers/users_controller.rb]: you're right. Thanks!

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -1,49 +1,84 @@
...
     <%= f.password_field :password_confirmation %>
 
-    <div id="terms_and_conditions_box" >
+    <!-- Should make "other" a free form text -->
+    <%= f.label :gender %>
+    <%= f.select :gender, ["Male", "Female", "Other"] , prompt: "Choose..." %>
+
+  
+    <%= f.label :studentID , "Student Number (if applicable)" %>
+    <%= f.text_field :studentID, autofocus: true %>
```
Comment (tahnok) [inline app/views/users/new.html.erb]: You might have to change this to `student_id`

**Comment by tahnok** [comment]
Also it looks like there are some legitimate test failures from CI!

File: app/mailers/msr_mailer.rb [hunk 3]
```diff
@@ -1,14 +1,20 @@
 class MsrMailer < ApplicationMailer
+	# **Have to check this**
```
Comment (tahnok) [inline app/mailers/msr_mailer.rb]: extra comment?

Comment (Parastoo-S) [inline app/mailers/msr_mailer.rb]: yea I'll clean it up. Also I won't merge this rn cus Zaineb said some things need to change and I will make the changed and merge them all together.

**Pull request merged**